backbone.js

```diff
@@ -979,14 +979,14 @@
     sort: function(options) {
       if (!this.comparator) throw new Error('Cannot sort a set without a comparator');
       options || (options = {});
+      var comparator = _.isFunction(this.comparator) ? _.bind(this.comparator, this) : this.comparator;
```

Comment:
Because `iteratee` methods no longer have a `context` param punting for devs to just pass in iteratees pre-bound with `_.bind` or `Function#bind`.